index.html

```diff
@@ -36,6 +36,7 @@
             <div id="install">
                 <ul class="commands commands-tooltip">
                     <li>cd /boot/EFI/refind/</li>
+                    <li>mkdir -p themes && cd themes</li>
                     <li>git clone {{site.github.repository_url}}</li>
                     <li>echo "include refind-minimal/theme.conf" >> refind.conf</li>
```

Comment:
Sorry about that, rushed comment. This path doesent included the themes directory.